docs/content/stable/explore/change-data-capture/debezium-connector-yugabytedb.md

```diff
@@ -932,8 +1037,14 @@ We use a custom record extractor (`YBExtractNewRecordState`) so that the sinks u
 See the following section for more details on `YBExtractNewRecordState`.
 {{< /note >}}
 
+### Transformers
```

Comment:
on line 985 - the link needs to be an absolute path because integrations is only in preview - preview/explore/change-data-capture/debezium-connector-yugabytedb/#deployment